hail/python/test/hail/backend/test_service_backend.py

```diff
@@ -1,8 +1,13 @@
+import os
+
 import hail as hl
 
 from ..helpers import skip_unless_service_backend
 from hail.backend.service_backend import ServiceBackend
 
+CLOUD = os.environ.get('HAIL_CLOUD')
```

Comment:
This is going to treat `None` and Azure the same. Can we keep this as `os.environ['HAIL_CLOUD']` but just put this line inside the relevant test?